src/js/containers/search/SearchSidebarSubmitContainer.jsx

```diff
@@ -120,7 +123,13 @@ export default connect(
         stagedFilters: state.filters,
         appliedFilters: state.appliedFilters.filters
     }),
-    (dispatch) => bindActionCreators(combinedActions, dispatch)
+    (dispatch) => ({
+        ...bindActionCreators(combinedActions, dispatch),
+        resetNaicsTree: () => {
```

Comment:
Do we want this resetNaicsTree here or should it be moved out to it's own component?

Reply:
Not sure what you mean here. Feel free to HMU on slack for a call if you wanna talk about it!

Reply:
Yeah, I wrote that kinda fast.  Wouldn't we want `resetNaicsTree` in its own file or with the other naics actions in `redux/actions/`?

Or just use `setChecked` or `setUnchecked`